### ISSUE TITLE
Extra cron entries as per documentation

### ISSUE THREAD

**Issue by PVince81** [owncloud-ansible/owncloud#11]
Additionally to `occ system:cron`, there are other jobs that are recommended to be added as separate system cron entries:

See https://doc.owncloud.com/server/10.1/admin_manual/configuration/server/background_jobs_configuration.html#available-background-jobs

These occ commands require extra cron jobs in the system crontab, where the interval could be different. Maybe we could add extra variables for each of these jobs and a configurable interval ?

**Comment by xoxys** [comment]
We can add it if there are jobs which should be configured on (nearly) every oC instance. Otherwise, the job should not be part of the default configuration but can added to the inventory group_vars for this specific setup.

@PVince81 Would be cool if you can specify which jobs are really necessary and recommended. For me it's currently a bit unclear. The description for most of the other documented cronjobs sounds like that it really depends on the end user environment.

**Comment by PVince81** [comment]
it really does depend on the end user environment

not sure if @cdamken or @voroyam have some generic recommendations for those

**Comment by enbrnz** [comment]
@cdamken Can we have a discussion about which additional cron jobs are absolutely necessary in a production instance by default?

**Comment by enbrnz** [comment]
One such job should be perhaps daily(?):
```
occ dav:cleanup-chunks
```
As when you have authenticated users, they'll upload using chunking and therefore when a download fails those won't ever get cleaned, as far as I understand.

**Comment by cdamken** [comment]
Yes, an example of the jobs is in Damken Cloud, I will provide you later

**Comment by cdamken** [comment]
here is an example of the cron jobs:

```
*/5  *  *  *  * /usr/bin/php -f /var/www/owncloud/occ system:cron
8 0 * * * /usr/bin/php -f /var/www/owncloud/occ dav:cleanup-chunks
13 0 * * * /usr/bin/php -f /var/www/owncloud/occ dav:sync-system-addressbook
18 0 * * * /usr/bin/php -f /var/www/owncloud/occ federation:sync-addressbooks
23 0 * * * /usr/bin/php -f /var/www/owncloud/occ trashbin:expire
28 0 * * * /usr/bin/php -f /var/www/owncloud/occ versions:expire
```

I would not recommend having `auto` configuration in versions and trashbin otherwise the last 2 don't work correctly
